libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -9,16 +9,18 @@ import {
 } from '@tmo/books/data-access';
 import { FormBuilder } from '@angular/forms';
 import { Book } from '@tmo/shared/models';
-import { Observable } from 'rxjs';
+import { Observable, Subject } from 'rxjs';
+import { debounceTime, distinctUntilChanged, takeUntil } from 'rxjs/operators';
 
 @Component({
   selector: 'tmo-book-search',
   templateUrl: './book-search.component.html',
   styleUrls: ['./book-search.component.scss']
 })
-export class BookSearchComponent {
+export class BookSearchComponent implements OnDestroy {
   books$: Observable<ReadingListBook[]> = this.store.select(getAllBooks);
-  ;
+  searchTermSubject = new Subject<string>();
+  unsubscribeSubject = new Subject<any>();
```

Comment:
Generally prefer to mark streams with the `$` suffix. Providing the type as the suffix like `Subject` is probably overkill. It's more useful to just define the purpose of the stream.

You also don't need to provide the type - you can use `Subject()` and then simply `unsubscribeSubject$.next()`

Reply:
Updated changes.